python3-dbus-python.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "python3-dbus-python",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"dbus-python\" --no-build-isolation"
+    ],
+    "build-options": {
+        "env": {
+            "LDFLAGS": "-lm",
+            "CFLAGS": "-lm"
+        }
+    },
```

Comment:
is this needed?

Reply:
It _was_ required when I still used a custom Python installation. No as I used the runtime's one, I can remove it.